src/jsifier.mjs

```diff
@@ -601,15 +601,7 @@ function(${args}) {
             warn('To build in STANDALONE_WASM mode without a main(), use emcc --no-entry');
           }
         }
-        if (!RELOCATABLE) {
```

Comment:
OTOH, "not relocatable == normal", and it's nice to have the normal path first? Then reading the more complex relocatable one after is easier.

Reply:
Actually in this case I like the `if RELOCATABLE`.

The code here is part of the undefined symbol handling.   Its within a block and its already an error in most cases.

It reads like this:

```
if (is_undefined) {
   maybe_report_an_error();
   // Here we are continuing with the undefined symbol
   if (RELOCATABLE) {  // AKA dynamic linking
      create_a_dylink_stub();
   } else {
      create_an_abort_stub();
   } 
}
```

I think this is more logical that the other way around:

```
if (is_undefined) {
   maybe_report_an_error();
   // Here we are continuing with the undefined symbol
   if (!RELOCATABLE) {
      create_an_abort_stub();
   } else {
      create_a_dylink_stub();
   } 
}
```

Reply:
Fair enough, I can live with this one. Reading the others again now.